src/harness/security_testcase.py

```diff
@@ -261,7 +261,6 @@ def assertTlsHandshakeFailureOrHttp403(self, client_cert, client_key, ciphers=No
     try:
       self.assertTlsHandshakeFailure(client_cert, client_key, ciphers, ssl_method)
     except AssertionError as e:
-      self.SasReset()
```

Comment:
why are we deleting this line ?

Reply:
because "SasReset ()" is already called by the SSS_xx.testcase.py that use "assertTlsHandshakeFailureOrHttp403" so we do not need to reset sas again